samples/circuitbreaker/circuitbreaker.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: httpbin-destination-rule
+spec:
+  host: httpbin
+  trafficPolicy:
+    portLevelSettings:
+    - port:
+        number: 8000
+      connectionPool:
+        http:
+          http1MaxPendingRequests: 1
+          maxRequestsPerConnection: 1
```

Comment:
IIUC these http setting does not take effect

Reply:
yes, just a place holder.